docs/insights/coverage.md

```diff
@@ -0,0 +1,63 @@
+---
+id: coverage
+title: Measuring Your Test Coverage
+sidebar_label: Coverage
+description: See a breakdown of the browser and device variations covered by your tests.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Coverage report allows Org Admins to view and download the breakdown of browsers and mobile devices the tests across their organization have run against.
```

Comment:
Can we add here one or two sentences describing the value of this tool? Not sure if the docs are the place for it. But if we do something along the lines of:
- make better decisions based on data 
- use data to inform your testing strategy and coverage based on your market needs

Maybe Alistair can shed some more light on the value proposition?